app/gcl/cloud-digital-leader/page.tsx

```
@@ -2354,50 +2354,50 @@ function Section8() {
+                        <span className="hero-badge">
+                            <span className="dot dot-yellow" />
+                            50–60問
+                        </span>
+                        <span className="hero-badge">
+                            <span className="dot dot-green" />
+                            推奨経験 6ヶ月+
+                        </span>
+                    </div>
+                </section>
+
+                <nav className="snav" role="navigation" aria-label="セクションナビゲーション">
```

Biome (2.4.15)

[error] Using the role attribute 'navigation' on the 'nav' element is redundant, because it is implied by its semantic.

(lint/a11y/noRedundantRoles)